gazebo_plugins/include/gazebo_plugins/gazebo_ros_joint_effort.hpp

```diff
@@ -0,0 +1,79 @@
+// Copyright 2012 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAZEBO_PLUGINS__GAZEBO_ROS_JOINT_EFFORT_HPP_
+#define GAZEBO_PLUGINS__GAZEBO_ROS_JOINT_EFFORT_HPP_
+
+#include <gazebo/common/Plugin.hh>
+#include <sensor_msgs/msg/joint_state.hpp>
+
+#include <memory>
+
+namespace gazebo_plugins
+{
+class GazeboRosJointEffortPrivate;
+
+/// This plugin collects data from a ROS topic and applies effort to a joint accordingly.
+/**
+  \details The last received effort will be continuously added to the joint at every simulation iteration.
+  Send an empty / zero message to stop applying a effort.
+
+  Example Usage:
+  \code{.xml}
+    <plugin name="gazebo_ros_effort" filename="libgazebo_ros_effort.so">
+
+      <ros>
+
+        <!-- Add a namespace -->
+        <namespace>/test</namespace>
+
+        <!-- Remap the default topic -->
+        <argument>gazebo_ros_effort:=effort_test</argument>
```

Comment:
```suggestion
        <argument>joint_efforts:=effort_test</argument>
```

I think it needs to be like this for proper remapping